inference-engine/src/transformations/src/transformations/softplus_to_mish_fusion.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/softplus_to_mish_fusion.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::pass::SoftPlusToMishFusion::SoftPlusToMishFusion() {
+    auto input = ngraph::pattern::any_input();
+    auto softplus = std::make_shared<ngraph::opset4::SoftPlus>(input);
```

Comment:
Softplus and Tanh could have more than one consumer and this replacement can break something. So I suggest to use wrap_type and use predicate that checks consumers count